automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/security/AuthDomainWorkspaceSpec.scala

```diff
@@ -408,6 +407,33 @@ class AuthDomainWorkspaceSpec extends FreeSpec /*with ParallelTestExecution*/ wi
             }
           }
         }
+
+        "when the user is a billing project owner" - {
+          "can be seen but is not accessible" in {
+            val user = UserPool.chooseProjectOwner
+            implicit val authToken: AuthToken = authTokenDefault
+            withGroup("AuthDomain") { authDomainName =>
+              withGroup("AuthDomain") { authDomain2Name =>
+                withCleanBillingProject(user, userEmails = List(defaultUser.email)) { projectName =>
+                  withWorkspace(projectName, "AuthDomainSpec_reject", Set(authDomainName, authDomain2Name)) { workspaceName =>
```

Comment:
Since this is in the "when shared with them" block, there should be an additional argument to `withWorkspace`.
```
withWorkspace(projectName, "AuthDomainSpec_reject", Set(authDomainName, authDomain2Name), List(AclEntry(user.email, WorkspaceAccessLevel.Reader)))
```
Also, I think the workspace should be owned by some other user in order to make sure we're testing the access path as billing project owner. @MatthewBemis do you agree?

Reply:
`withCleanBillingProject` owned by `user`. User's AuthToken created in method.
```
 def withCleanBillingProject(newOwnerCreds: Credentials, ownerEmails: List[String] = List(), userEmails: List[String] = List())(testCode: (String) => Any): Unit = {
    withCleanBillingProject(newOwnerCreds.email, ownerEmails, userEmails)(newOwnerCreds.makeAuthToken _)(testCode)
  }
```

Reply:
this is one of many examples of why implicit auth tokens suck. the workspace is owned by "authTokenDefault" and the project is owned by "user" with the defaultUser as a _user_

Reply:
Oh, you're absolutely right. I thought I had read that close enough but obviously not. Workspace ownership is fine.

I still think the workspace should be shared with `user` to match the test description.

Reply:
are you talking about the "when shared with them" on line 382? this is outside of that block

Reply:
🤦‍♂️ You're right again. I'm going sneak away now as if I never said anything.

Reply:
@breilly2 your questions led to discovery of a hole in test coverage. thank you.
@MatthewBemis thank you guys.